server/app/services/geo/esri/EsriClient.java

```diff
@@ -144,11 +144,28 @@ CompletionStage<Optional<JsonNode>> fetchAddressSuggestions(ObjectNode addressJs
         Optional.ofNullable(addressJson.findPath(AddressField.STATE.getValue()).textValue());
     Optional<String> postal =
         Optional.ofNullable(addressJson.findPath(AddressField.ZIP.getValue()).textValue());
-    address.ifPresent(val -> request.addQueryParameter("address", val));
-    address2.ifPresent(val -> request.addQueryParameter("address2", val));
-    city.ifPresent(val -> request.addQueryParameter("city", val));
-    region.ifPresent(val -> request.addQueryParameter("region", val));
-    postal.ifPresent(val -> request.addQueryParameter("postal", val));
+    String singleLineAddress = "";
```

Comment:
nonblocking: consider switching to StringBuilder

Reply:
Good call. Updated!